tasks/tos.yml

```diff
@@ -0,0 +1,16 @@
+- name: Check if `conda tos` is available
+  ansible.builtin.command: "{{ miniconda_prefix }}/bin/{{ miniconda_executable }} tos --help"
```

Comment:
Hmmmm the return code of `mamba tos --help` is `0` so this does not allow us to check whether the `tos` command is available or not. Since `miniconda_tos_channels` is empty by default I guess it is fine? Or is there another way to check that  the `tos` command is available?